Assignment-2/SqlConnect.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ *  Class to establish connection with MySQL and perform insert operations to tables.
+ */
+class SqlConnect {
+  private $conn;
+
+  /**
+   * Constructor function to establish connection with database in MySQL.
+   *
+   * @param string $servername
+   *   Server name of MySQL.
+   *
```

Comment:
No need of empty lines between multiple params.
An empty line should be present between params and return value.